code/controllers/subsystem/fire_burning.dm

```diff
@@ -27,10 +27,11 @@ SUBSYSTEM_DEF(fire_burning)
 				return
 			continue
 
-		if(O.resistance_flags & ON_FIRE)
```

Comment:
where was the redundancy for this?

Reply:
Just realised it's in there for if something is extinguished mid-run, will put it back